tick/eval/types.go

```diff
@@ -0,0 +1,189 @@
+package eval
+
+import (
+	"reflect"
+	"regexp"
+
+	"github.com/influxdata/kapacitor/tick"
+)
+
+type ValueType uint8
+
+const (
+	InvalidType ValueType = iota << 1
+	TFloat64
+	TInt64
+	TString
+	TBool
+	TRegex
+)
+
+const TNumeric = TFloat64 & TInt64
+
+func isNumericValueType(valueType ValueType) bool {
+	return (valueType|TInt64 == TInt64) || (valueType|TFloat64 == TFloat64)
+}
+
+var valueTypeToName = map[ValueType]string{
```

Comment:
This should also be converted to  method on the `ValueType` as the `String` method with a static switch statement. Something like this https://github.com/influxdata/kapacitor/blob/master/alert.go#L46 except don't panic. 

*Note to self  remove panic in alert.go*